src/components/calcite-alert/calcite-alert.e2e.ts

```diff
@@ -44,9 +41,28 @@ describe("calcite-alert", () => {
     expect(close).toBeNull();
     expect(icon).toBeNull();
   });
-});
 
-describe("calcite-alert", () => {
+  it("forces close button when dismiss and link are present", async () => {
```

Comment:
this description could use some love - I'm not quite sure what's being tested here. My guess is the close button shouold be present and visible when these two props are set.